src/internal/modules/returns-notifications/lib/reducer.js

```diff
@@ -77,6 +77,9 @@ const mapLicencesToState = (licences, refDate) => {
     licenceRow.documents.forEach(documentRow =>
       acc.set(documentRow.document.id, mapDocumentRow(licenceRow, documentRow, refDate))
     );
+    if (licenceRow.documents.length === 0) {
```

Comment:
This is interesting. Does this cater for an edge case? If so, it might be worth putting an inline comment explaining why? (It's not a problem - just curious)